### PR TITLE
Milestone: Contact validation

### PR DESCRIPTION
**Pull request by iambenkis**
# validate contact form
  # The contact-validation  branch contains code for the form validation. 
# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] The project sticks to the Figma template
- [ ] If the validation is OK, the form will be sent. 
- [ ] If the validation is not OK, you show an error message  ````Kindly check your email```` and the form will be not sent.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -436,6 +436,28 @@ button:hover {
...
+  font-size: 80%;
+  color: rgb(219, 0, 0); 
+  border-radius: 0 0 5px 5px;
+  box-sizing: border-box;
+  position: absolute;
+  top: 88%;
+  left: 50%; 
+  border-radius: 2px;
+  font-size: .9rem;
+}
```
Comment (1GORDON) [inline style.css]: Great job team 🤝

- [ ] [Optional] I would recommend the error message be more explicit. Something like `Please make email lowercase`.

File: index.html [hunk 2]
```diff
@@ -266,7 +277,8 @@ <h2 class="contact-me-text">Contact me</h2>
                             cols="30"
                             required>
                         </textarea> 
-                        <button type="submit">Get in Touch</button>    
+                        <span class="error" aria-live="polite"></span> 
+                        <button type="submit" id="submit">Get in Touch</button>
```
Comment (1GORDON) [inline index.html:283]: Great job team 🤝

- [ ] As specified in the project requirements, you may kindly align your button accordingly. Please check the design provided in the Figma template.
- Your design
align button

- Figma design
Figma design

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @iambenkis & @DebbyMiressa ✋

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Linters are passing ✔️
- Professional Readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

_Check the comments under the review._

## Optional suggestions

- [ ] [Optional] You may kindly remove all commented code from your style sheet. Please check line `536`. Leaving commented code in your project is not regarded as a good practice.
- [ ] [Optional] You may as well remove commented code from your HTML file. Check out line `288`.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @iambenkis               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

- [ ] [OPTIONAL] - Kindly remove the commented out CSS code in style.css lines number 541 - 543.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**